irr/src/CMakeLists.txt

```diff
@@ -3,8 +3,13 @@
 if(NOT APPLE)
 	set(DEFAULT_SDL2 ON)
 endif()
+if(APPLE AND CMAKE_SYSTEM_NAME STREQUAL "iOS")
+	set(DEFAULT_SDL2 ON)
+	set(DEFAULT_ANGLE ON)
+endif()
 
 option(USE_SDL2 "Use the SDL2 backend" ${DEFAULT_SDL2})
+option(USE_ANGLE "Use the Google ANGLE EGL and OpenGL ES" ${DEFAULT_ANGLE})
```

Comment:
and I would also make this a hidden option

Reply:
I make it an available option for macOS build also, because of #16616.

So I think it does not need to be hidden.